spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/properties/RateLimitProperties.java

```diff
@@ -64,6 +67,21 @@
     @Valid
     @NotNull
     private Repository repository = Repository.IN_MEMORY;
+    private int postFilterOrder = SEND_RESPONSE_FILTER_ORDER - 10;
+    private int preFilterOrder = FORM_BODY_WRAPPER_FILTER_ORDER;
+
+    @Override
+    public void afterPropertiesSet() {
```

Comment:
Use validator instead (like @NotNull)

Reply:
In this particular case the validation is made against two fields, is it possible to make it using those annotations? I mean, without creating a custom one to make the validation. 

@lchayoun

Reply:
please check how it's done in `ResourceServerProperties `. i'm going to use the same approach if you approve (although it looks bulky as for me)

Reply:
Sounds good to me, WDYT @lchayoun?